### PR TITLE
Create cartago.json

### PR DESCRIPTION
**Pull request by crebollobr**
Hello, 

This data is a filter of cicy Cartago of DANE data. If the data is ok I will put the rest of data.

https://github.com/digital-guard/preserv-CO/tree/main/data/_pk0006.02

Thanks

### CODE REVIEW

File: sources/co/vac/cartago.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "addresses": [
+            {
+                "name": "city",
+                "data": "https://addressforall-my.sharepoint.com/personal/operacao_addressforall_org/_layouts/15/download.aspx?share=EdyPQRg2TShKgpnV5DD2Y0oBZbdPpsjUcI0qVczflqdwOQ",
+                "website": "https://github.com/digital-guard/preserv-CO/tree/main/data/_pk0006.02",
+                "protocol": "http",
+                "license": {
+                    "url": "https://creativecommons.org/licenses/by/4.0/",
+                    "attribution": true,
+                    "attribution name": "CC-BY-4.0",
```
Comment (iandees) [inline sources/co/vac/cartago.json:22]: We need to update this with the correct attribution information, not the license name.

File: sources/co/vac/cartago.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "schema": 2,
+    "layers": {
+        "addresses": [
+            {
+                "name": "city",
+                "data": "https://addressforall-my.sharepoint.com/personal/operacao_addressforall_org/_layouts/15/download.aspx?share=EdyPQRg2TShKgpnV5DD2Y0oBZbdPpsjUcI0qVczflqdwOQ",
+                "website": "https://github.com/digital-guard/preserv-CO/tree/main/data/_pk0006.02",
+                "protocol": "http",
+                "license": {
+                    "text": "CC-BY-4.0R",
```
Comment (iandees) [inline sources/co/vac/cartago.json:20]: I don't think `CC-BY-4.0R` exists. Is this a typo?

Comment (crebollobr) [reply sources/co/vac/cartago.json]: I will fix

**Comment by iandees** [comment]
Uploading the file as downloaded from Sharepoint, since the Sharepoint URL doesn't seem to give valid data outside of a browser.

[co202307_cartago_ghs.geojson.zip](https://github.com/openaddresses/openaddresses/files/15172276/co202307_cartago_ghs.geojson.zip)

**Comment by iandees** [comment]
Uploading again because macOS's compression tool added another file inside the zip.

[co202307_cartago_ghs.geojson.zip](https://github.com/openaddresses/openaddresses/files/15172322/co202307_cartago_ghs.geojson.zip)

**Comment by iandees** [comment]
Looks good!

**Pull request merged**